floor_generator/lib/writer/query_method_writer.dart

```diff
@@ -76,36 +68,40 @@ class QueryMethodWriter implements Writer {
 
   @nonNull
   List<String> _generateInClauseValueLists() {
-    var index = 0;
     return _queryMethod.parameters
-        .map((parameter) {
-          if (parameter.type.isDartCoreList) {
-            index++;
-            return '''final valueList$index = ${parameter.displayName}.map((value) => "'\$value'").join(', ');''';
-          } else {
-            return null;
-          }
-        })
-        .where((string) => string != null)
-        .toList();
+        .where((parameter) => parameter.type.isDartCoreList)
+        .mapIndexed((index, parameter) {
+      // TODO #165 what about type converters that map between e.g. string and list?
```

Comment:
I think we can leave that out for now and create a new issue for that to discuss the possibilities. Making a wrapper method which converts this is a simple thing for a workaround for now.

Reply:
Will be solved in #403.